tests/meta.js

```diff
@@ -6,20 +6,19 @@ import process from 'child_process';
 import Promise from 'bluebird';
 
 test('get packages', t => {
```

Comment:
This is an NPM issue, was [resolved in3.5.3](https://github.com/npm/npm/issues/11231)